lumigator/python/mzai/backend/backend/main.py

```diff
@@ -67,6 +67,13 @@ def create_app() -> FastAPI:
 
     app.include_router(api_router)
 
+    if os.environ.get("DEBUG"):
+        import debugpy  # type: ignore
+
+        debug_port = os.environ.get("DEBUG_PORT", 5678)
+        logger.info(f"Starting debugpy on port {debug_port}")
+        debugpy.listen(("0.0.0.0", debug_port))
```

Comment:
Let’s avoid hardcoding the host and port. Perhaps backend.settings is a better place to define sensible defaults and read them from there.